src/chat-bubble/interfaces.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export interface ChatBubbleProps {
+  /** Avatar slot paired with the chat bubble. */
+  avatar: React.ReactNode;
+
+  /** Defines the type of the chat bubble and sets its color accordingly.  */
+  type: ChatBubbleProps.Type;
+
+  /** Content of the chat bubble */
+  children: React.ReactNode;
+
+  /** Actions slot of the chat bubble, placed at the bottom left. */
+  actions?: React.ReactNode;
+
+  /**
+   * Adds a loading bar to the bottom of the chat bubble. This property should only be used for Generative AI loading state.
+   * If avatar is being used, set its `loading` state to true.
+   */
+  isGeneratingContent?: boolean;
+
+  /** Adds aria-label to the chat bubble container. Use this to provide a unique accessible name for each chat bubble on the page. */
```

Comment:
Should we recommend using a timestamp here?

Reply:
All playground examples are using timestamps, but I can add here as well